docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -21,7 +21,7 @@ For Elastic Stack upgrade, check <<{p}-upgrading-stack,Upgrade the Elastic Stack
 
 ECK reached general availability (GA) status with the link:https://www.elastic.co/blog/elastic-cloud-on-kubernetes-ECK-is-now-generally-available[release of version 1.0.0]. The latest available GA version is {eck_version}. It is compatible with the previous GA releases (1.0.x and higher) and the beta release (1.0.0-beta1), and can be upgraded in-place (<<{p}-ga-openshift, with a few exceptions>>) by applying the new set of deployment manifests. Previous alpha releases, up to and including version 0.9.0, are not compatible with the GA and beta releases and link:https://www.elastic.co/guide/en/cloud-on-k8s/1.0-beta/k8s-upgrading-eck.html[require extra work to upgrade].
 
-Before upgrading, refer to the <<release-notes-{eck_version}, release notes>> to make sure that the release does not contain any breaking changes that could affect you. The <<release-highlights-{eck_version},release highlights document>> provides more details and possible workarounds for any breaking changes or known issues in each release.
+Before upgrading, refer to the <<release-notes-{eck_version}, release notes>> to make sure that the release does not contain any breaking changes that could affect you. The <<release-notes-{eck_version},release notes>> also provide more details and possible workarounds for any breaking changes or known issues in each release.
```

Comment:
I think it was on purpose to link the highlights here, especially for known issues in the release (ex: [1.7.0](https://www.elastic.co/guide/en/cloud-on-k8s/master/release-highlights-1.7.0.html#k8s-170-known-issues), [1.5.0](https://www.elastic.co/guide/en/cloud-on-k8s/master/release-highlights-1.5.0.html#k8s-150-known-issues)).
But I see your point as the 1.9.0 release notes explain workarounds for the breaking changes.
Maybe we should rephrase the first sentence to keep the highlights link?
```suggestion
Before upgrading, refer to the <<release-highlights-{eck_version},release highlights document>> to see the new and notable changes and known issues in each release and make sure that the release does not contain any breaking changes that could affect you. The <<release-notes-{eck_version},release notes>> also provide more details and possible workarounds for any breaking changes.
```

-

Reply:
Added an issue - https://github.com/elastic/cloud-on-k8s/issues/5111.